grpc-utils/src/main/java/com/glegoux/grpc/GrpcServerSimple.java

```diff
@@ -19,16 +21,29 @@ public class GrpcServerSimple implements GrpcServer {
     private Server server;
     private HTTPServer monitoringServer;
 
+    private final List<BindableService> services;
+
+
+    public GrpcServerSimple(List<BindableService> services) {
+        this.services = services;
+    }
+
+    @Override
     public void start(int port, int monitoringPort) throws IOException {
 
```

Comment:
actually all the job done in the `start` method could be done in the constructor. And the `start` method could just start the service.

Reply:
Fone